api/access_key.go

```diff
@@ -24,7 +24,7 @@ type ListAccessKeysRequest struct {
 }
 
 func (r ListAccessKeysRequest) ValidationRules() []validate.ValidationRule {
-	return r.PaginationRequest.ValidationRules()
+	return nil
```

Comment:
I'd like to see some comments here that explain why this is needed for future reference